app/src/main/java/com/example/bookclub/LibraryActivity.java

```diff
@@ -1,14 +1,82 @@
 package com.example.bookclub;
 
 import androidx.appcompat.app.AppCompatActivity;
+import androidx.recyclerview.widget.ItemTouchHelper;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
 
 import android.os.Bundle;
+import android.util.Log;
+
+import com.example.bookclub.Adapters.LibraryItemAdapter;
+import com.example.bookclub.models.LibraryItem;
+import com.example.bookclub.utils.BookListItemHelper;
+
+import com.parse.GetCallback;
+import com.parse.ParseException;
+import com.parse.ParseQuery;
+import com.parse.ParseUser;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+import java.util.List;
 
 public class LibraryActivity extends AppCompatActivity {
+    private RecyclerView rvLibraryItem;
+    protected LibraryItemAdapter mLibraryItemAdapter;
+    protected List<LibraryItem> mAllLibraryItems;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_library);
+        setContentView(R.layout.library_item);
+
+        rvLibraryItem = findViewById(R.id.rvLibraryItem);
+        mAllLibraryItems = new ArrayList<>();
+        mLibraryItemAdapter = new LibraryItemAdapter(this, mAllLibraryItems);
+        ItemTouchHelper.Callback callback = new BookListItemHelper(mLibraryItemAdapter);
+        ItemTouchHelper itemTouchHelper = new ItemTouchHelper(callback);
+        mLibraryItemAdapter.setTouchHelper(itemTouchHelper);
+        itemTouchHelper.attachToRecyclerView(rvLibraryItem);
+        rvLibraryItem.setAdapter(mLibraryItemAdapter);
+        rvLibraryItem.setLayoutManager(new LinearLayoutManager(this));
+        populateLibraryItems();
+    }
+
+    private void populateLibraryItems() {
```

Comment:
This loading code would do well in its own utility method meant for parsing the api fetches.